raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -701,22 +702,28 @@ def events_for_expired_locks(
 
         secrethash = mediator_state.secrethash
         locked_lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
-        lock_expired = channel.is_lock_expired(
-            end_state=channel_state.our_state,
-            locked_lock=locked_lock,
-            secrethash=secrethash,
-            block_number=block_number,
-        )
-        if locked_lock and lock_expired:
-            # Lock has expired, cleanup...
-            transfer_pair.payee_state = 'payee_expired'
-            expired_lock_events = channel.events_for_expired_lock(
-                channel_state,
-                secrethash,
-                locked_lock,
-                pseudo_random_generator,
+
+        if locked_lock:
+            lock_expiration_threshold = (
+                locked_lock.expiration +
+                DEFAULT_NUMBER_OF_CONFIRMATIONS_BLOCK * 2
```

Comment:
Should we define `LOCK_EXPIRATION_THRESHOLD = DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS * 2` to prevent us from re-doing this calculation all over?

Reply:
that is why I had the function `lock_expiration_threshold`, but the threshold for the senders is different than the one of the receivers, and I didn't get a good idea of how to name it.

i wouldn't introduce another constants for this, since the goal is to actually remove the constant and make it dependent on the chain